core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -556,8 +557,18 @@ private void setupServerMetrics() {
                                 ConnectionLimitingHandler::numConnections);
         }
 
+        private void scheduleExceptionReporting() {
+            final ServerErrorHandler serverErrorHandler = config().errorHandler();
+            if (!config().unloggedExceptionReportInterval().isZero() &&
+                serverErrorHandler instanceof ExceptionReportingServerErrorHandler) {
+                ((ExceptionReportingServerErrorHandler) serverErrorHandler).start(config.workerGroup());
+            }
+        }
+
         @Override
         protected CompletionStage<Void> doStop(@Nullable Void arg) {
+            unscheduleExceptionReporting();
```

Comment:
Just stop?
```suggestion
            stopExceptionReporting();
```

Reply:
I'll use `ServerListener` to invoke above methods as mentioned by @ikhoon 
https://github.com/line/armeria/pull/4687/files/43095fbe053521709df01fe8996ee1d0bdbe8df9#r1116797134